pytorch_lightning/plugins/environments/cluster_environment.py

```diff
@@ -37,13 +43,15 @@ def creates_children(self) -> bool:
         )
         return self.creates_processes_externally
 
+    @property
```

Comment:
You are using this PR to add these missed property decorators. Is this something that should go into the bug-fix release? If yes, you'll need to split this PR

Reply:
No, I don't think I'm adding any missing ones.
They never were properties. I'm combining the renaming + the property change, as it is a breaking change anyway.

The PR title is misleading, changing it.

Reply:
should these be properties or static methods?

Reply:
there are plugins like the LSF that save some state and read it. So static would not work there.
I think if we want these plugins be customizable, non-static is the most flexible. I don't think there is a cost associated with it not to do it. 

Property or method, I don't know, don't care. I only do it because when I added a set_x() method in this PR #6941, people requested me to convert it to properties and setters (https://github.com/PyTorchLightning/pytorch-lightning/pull/6941#pullrequestreview-633644884). Then I created this issue here #6303.